reference/fleet/migrate-elastic-agent.md

```diff
@@ -210,7 +208,11 @@ The agents can now be re-enrolled into these policies and migrated over to the n
 
 ## Migrate {{agent}}s to the new target cluster [migrate-elastic-agent-migrated-agents]
 
-In order to ensure that all required API keys are correctly created, the agents in your current cluster need to be re-enrolled into the new, target cluster.
+::::{note}
+Agents to be migrated cannot be tamper-protected or running as a {{fleet}} server.
```

Comment:
The official format is Fleet Server. You can use that directly or use the `fleet-server` sub.

```suggestion
Agents to be migrated cannot be tamper-protected or running as a {{fleet-server}}.
```